includes/class-wc-google-analytics-js.php

```diff
@@ -256,41 +260,39 @@ protected static function load_analytics_universal( $logged_in ) {
 			$ga_snippet_developer_id = '';
 		}
 
-		$ga_snippet_require      = $support_display_advertising .
+		$ga_snippet_require = $support_display_advertising .
 			$support_enhanced_link_attribution .
 			$anonymize_enabled .
-			$track_404_enabled . "
-		" . self::tracker_var() . "( 'set', 'dimension1', '" . $logged_in . "' );\n";
+			$track_404_enabled . '
+		' . self::tracker_var() . "( 'set', 'dimension1', '" . $logged_in . "' );\n";
 
 		if ( 'yes' === self::get( 'ga_enhanced_ecommerce_tracking_enabled' ) ) {
-			$ga_snippet_require .= "" . self::tracker_var() . "( 'require', 'ec' );";
+			$ga_snippet_require .= self::tracker_var() . "( 'require', 'ec' );";
 		} else {
-			$ga_snippet_require .= "" . self::tracker_var() . "( 'require', 'ecommerce', 'ecommerce.js');";
+			$ga_snippet_require .= self::tracker_var() . "( 'require', 'ecommerce', 'ecommerce.js');";
 		}
 
 		$ga_cross_domains = ! empty( self::get( 'ga_linker_cross_domains' ) ) ? array_map( 'esc_js', explode( ',', self::get( 'ga_linker_cross_domains' ) ) ) : false;
 
 		if ( $ga_cross_domains ) {
-			$ga_snippet_require .= '' . self::tracker_var() . "( 'require', 'linker' );";
-			$ga_snippet_require .= '' . self::tracker_var() . "( 'linker:autoLink', " . wp_json_encode( $ga_cross_domains ) . ');';
+			$ga_snippet_require .= self::tracker_var() . "( 'require', 'linker' );";
+			$ga_snippet_require .= self::tracker_var() . "( 'linker:autoLink', " . wp_json_encode( $ga_cross_domains ) . ');';
 		}
 
 		$ga_snippet_head         = apply_filters( 'woocommerce_ga_snippet_head', $ga_snippet_head );
 		$ga_snippet_create       = apply_filters( 'woocommerce_ga_snippet_create', $ga_snippet_create, $ga_id );
 		$ga_snippet_developer_id = apply_filters( 'woocommerce_ga_snippet_developer_id', $ga_snippet_developer_id );
 		$ga_snippet_require      = apply_filters( 'woocommerce_ga_snippet_require', $ga_snippet_require );
 
-		$code = "<script type='text/javascript'>" . $ga_snippet_head . $ga_snippet_create . $ga_snippet_developer_id . $ga_snippet_require . "</script>";
-		$code = apply_filters( 'woocommerce_ga_snippet_output', $code );
+		$code = $ga_snippet_head . $ga_snippet_create . $ga_snippet_developer_id . $ga_snippet_require;
 
-		return $code;
+		wc_enqueue_js( apply_filters( 'woocommerce_ga_snippet_output', $code ) );
```

Comment:
This changes the UA to the footer as well. Technically they have till June to upgrade to a GTag implementation, but we shouldn't be breaking any older sites. I'd prefer to revert these commits and leave these snippets as is. Same applies for the legacy Standard Tracking (function `load_analytics_classic`)

Blocks doesn't have compatibility for UA anyway so no need to resolve the conflict here.